### DISCUSSION TITLE
Having challenges w/ CapturePhotoAsync on android 33

### DISCUSSION THREAD

**Discussion by bradtwurst** [dotnet/maui#11491] [Q&A] [answered]
I am testing out a new maui app that is attempting to access the camera and capture the photo.

I am having an issue where there is an exception thrown when callling CapturePhotoAsync
that StorageWrite permission is required.

If I request that permission with the STORAGE_EXTERNAL_WRITE, it always returns false without any prompt.

The remote device in question that is having issues is a pixel 5 (api 33) connected via usb to VS2022.

The emulator is not having issues for the same level phone.

Another phone that is not at api 33 is also not having this issue.

Doing some spelunking, It appears that there was a change to the android api and 
there was a change that was done back in February by @jamesmontemagno 

https://github.com/dotnet/maui/pull/4697

that removed this line of code for the PickAsync() method

```
await Permissions.EnsureGrantedAsync<Permissions.StorageRead>();
```

---

Does a similar change need to be done for the CaptureAsync() method here

https://github.com/dotnet/maui/blob/6354e780d7738ef375447b901c0a583de3f4f0b3/src/Essentials/src/MediaPicker/MediaPicker.android.cs#L65

**Comment by berhir** [comment, marked as answer]
There is an issue for this bug: https://github.com/dotnet/maui/issues/11275

**Reply by bradtwurst** [reply]
https://github.com/dotnet/maui/issues/11275#issuecomment-1334910630

'downgrade' the api level to 32 until #11275 is addressed
